LIB386/LIB_MIDI/LIB_MIDI.H

```diff
@@ -1,10 +1,12 @@
 #ifdef	WATCOM9
-#pragma	library ("f:\projet\lib386\lib_midi\lb9_midi.lib");
+#pragma	library ("c:\lba\lib386\lib_midi\lb9_midi.lib");
 #else
-#pragma	library ("f:\projet\lib386\lib_midi\lib_midi.lib");
+#pragma	library ("c:\lba\lib386\lib_midi\lib_midi.lib");
 #endif
 
-#include "f:\projet\lib386\lib_midi\ail32.h"      // Audio Interface Library API function header
+// TODO
+// #include "lib_midi\ail32.h"      // Audio Interface Library API function header
+// #include "ail.h"      // Audio Interface Library API function header
```

Comment:
I would create a preprocessor macro to prevent those to be used during compilation instead of commenting the code.